docs/index.md

```diff
@@ -1,12 +1,14 @@
 ```{include} ../README.md
+:end-before: "## Capabilities"
```

Comment:
Does this mean it'll only copy the first header section from README? Neat!

Reply:
Yep, exactly.